Dockerfile.develop

```diff
@@ -15,9 +15,9 @@
 ###############################################################################
 # Create the develop, test, and build environment
 ###############################################################################
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7
 
-ARG GOLANG_VERSION=1.18.7
+ARG GOLANG_VERSION=1.19.3
```

Comment:
we should leave the Go version at 1.18 for the time being, but we can update to the latest service version `1.18.9`

https://go.dev/dl/go1.18.9.linux-amd64.tar.gz

Reply:
Will apply on the next commit.